src/Infrastructure/EntitySubscriber/ChannelPricingLogEntryEntitySubscriber.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\PriceHistoryPlugin\Infrastructure\EntitySubscriber;
```

Comment:
IMO it should be EventSubscriber.
```suggestion
namespace Sylius\PriceHistoryPlugin\Infrastructure\EventSubscriber;
```